From david-b@pacbell.net Wed Nov  7 13:31:47 2007
From: David Brownell <david-b@pacbell.net>
Date: Wed, 7 Nov 2007 12:59:54 -0800
Subject: USB: remove USB HUB entry from MAINTAINERS
To: Greg KH <greg@kroah.com>
Cc: linux-usb-devel@lists.sourceforge.net
Message-ID: <200711071259.55061.david-b@pacbell.net>
Content-Disposition: inline


Digging through old mail, I found a note about needing to remove the
separate entry for the USB HUB driver.  It's not been separable from
usbcore (host side!) since quite early in the 2.4 kernel series.
And Johanness certainly isn't involved with it any more.

Signed-off-by: David Brownell <dbrownell@users.sourceforge.net>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 MAINTAINERS |    7 -------
 1 file changed, 7 deletions(-)

--- a/MAINTAINERS
+++ b/MAINTAINERS
@@ -3864,13 +3864,6 @@ L:	linux-usb-devel@lists.sourceforge.net
 T:	git kernel.org:/pub/scm/linux/kernel/git/jikos/hid.git
 S:	Maintained
 
-USB HUB DRIVER
-P:	Johannes Erdfelt
-M:	johannes@erdfelt.com
-L:	linux-usb-users@lists.sourceforge.net
-L:	linux-usb-devel@lists.sourceforge.net
-S:	Maintained
-
 USB ISP116X DRIVER
 P:	Olav Kongas
 M:	ok@artecdesign.ee
